### ISSUE TITLE
Does it able to connect with llama for image understanding?

### ISSUE THREAD

**Issue by lucasjinreal** [Meituan-AutoML/VisionLLaMA#1]
Does it able to connect with llama for image understanding?

**Comment by cxxgtxy** [comment]
Yes. In principle, it can be trained using CLIP and replace the VIT in vision-language models such as MobileVLM(https://github.com/Meituan-AutoML/MobileVLM).
